internal/rbd/controllerserver.go

```diff
@@ -94,25 +94,52 @@ func (cs *ControllerServer) validateVolumeReq(ctx context.Context, req *csi.Crea
 	return nil
 }
 
+// isMultiWriterBlockRWOFile validates the volume capability slice against the access modes and access type.
+// if the capability is of multi write the first return value will be set to true and if the request
+// is of type block, the second return value will be set to true. If it is of type RWO and file mode, the third return
+// will be set to true. Otherwise, false values are returned.
+func isMultiWriterBlockRWOFile(caps []*csi.VolumeCapability) (bool, bool, bool, bool) {
```

Comment:
Move this function to csi-common utils.

Reply:
Four return variables is a LOT, please document them well.

Reply:
See #2717 for a similar change that helps with implementing CSI-Addons ReclaimSpace.

Reply:
rearranged this function to csi commons utils.